docs/modules/ROOT/pages/asciidoc-attributes.adoc

```diff
@@ -138,8 +145,8 @@ If an attribute matches a key in the theme file, the document attribute takes pr
 |Yes
 |`:scripts: cjk`
 
-|`text-align`^[7]^
-|xref:theme:text.adoc#text-align[Text alignment]
+|xref:theme:text.adoc#text-align[text-align] ^[7]^
+|`center` {vbar} `justify` {vbar} `left` {vbar} `right` {vbar} `inherit`
```

Comment:
I don't think this change is needed. When the value is a defined data type (in this case "Text alignment"), let's be consistent and just link to the page that defines the type. This is used throughout the theming documentation.